weave

```diff
@@ -168,6 +168,8 @@ exec_remote() {
         $(docker_run_options) \
         --pid=host \
         $(cni_volume_options "$@") \
+        -v /:/host \
+        -e HOST_ROOT="/host" \
```

Comment:
perhaps we should do this only for launch commands. Similar to what `cni_volume_options` does.